README.md

```diff
@@ -212,24 +207,21 @@ Row(
       style: TextStyle(fontSize: 43.0),
     ),
     const SizedBox(width: 20.0, height: 100.0),
-    AnimatedTextKit(
-      animatedTexts: [
-        RotateAnimatedText(
-          'AWESOME',
-          textStyle: rotateTextStyle,
-        ),
-        RotateAnimatedText(
-          'OPTIMISTIC',
-          textStyle: rotateTextStyle,
-        ),
-        RotateAnimatedText(
-          'DIFFERENT',
-          textStyle: rotateTextStyle,
-        ),
-      ]
-      onTap: () {
-        print("Tap Event");
-      },
+    DefaultTextStyle(
+      style: const TextStyle(
+        fontSize: 40.0,
+        fontFamily: 'Horizon',
+      ),
```

Comment:
I think it should work without wrapping it `DefaultTextStyle`?
Can you please verify?

Reply:
I just verified...It will work

Reply:
Yes, Flutter has an _implicit Default Text Style_ -- even when you don't use the widget.